rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :person, :book
+  attr_writer :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    @book.rentals.push(self)
+    @person.rentals.push(self)
```

Comment:
- [ ] Kindly consider removing `@` from `@book` and `@person`, this does not refer to the local variable